server/model/article.js

```diff
@@ -42,9 +42,15 @@ export class ArticleFactory {
     const contentType = getContentType(json.format);
     const bodyPartsRaw = bodyParser(articleBody);
     const standfirst = bodyPartsRaw.find(part => part.type === 'standfirst');
+    const mainComic: ?Picture = bodyPartsRaw[0] && bodyPartsRaw[0].type === 'picture' && contentType === 'comic' ? bodyPartsRaw[0].value : null;
+
+    if (mainComic) {
+      mainComic.isMain = true;
```

Comment:
From the [comment above](https://github.com/wellcometrust/wellcomecollection.org/pull/840/files#r113689341) - I'm not sure the is necessary?